mongodb/intro/README.md

```diff
@@ -19,6 +19,7 @@ sections:
     - arithmetic-aggregation-i
     - arithmetic-aggregation-ii
     - aggregation-iii
+    - aggregation-iv
```

Comment:
can we also rename these?

- aggregation-with-max-and-mind

Reply:
These are workout names, I can't rename a workout with 3-4 different operators with the name of 2. Did you mean the insights named: aggregation-with-accumulators-iii and iv?

Reply:
I've changed the names of the insights. As for workouts, they are all regarding multiple different operators, I can't name them according to one operator in the workout unless only that one is discussed there.